app/Repositories/Eloquents/BookUserEloquentRepository.php

```diff
@@ -69,4 +69,20 @@ public function getDataRequest($data = [], $with = [], $dataSelect = ['*'], $att
             ->orderBy($attribute[0], $attribute[1])
             ->paginate(config('view.paginate.book_request'));
     }
+
+    public function getDetailData($request)
+    {
+        try {
+            $with = [
+                'user',
+            ];
+
+            return $this->getData($request, $with)
+                ->pluck('user')
+                ->unique()
+                ->chunk(config('view.paginate.follow_user'));
+        } catch (Exception $e) {
+            return NULL;
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Generic.PHP.LowerCaseConstant.Found</br>**Severity**: error</br>**File**: app/Repositories/Eloquents/BookUserEloquentRepository.php</br>TRUE, FALSE and NULL must be lowercase; expected "null" but found "NULL"